app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,132 @@
+class TweetsController < ApplicationController
+  before_action :set_tweet, only: %i[ show edit update destroy ]
+
+  # GET /tweets or /tweets.json
+  def index
+    @tweet = Tweet.new
+    @tweets = Tweet.all.order(created_at: :asc)
+  end
+
+  # GET /tweets/1 or /tweets/1.json
+  def show
+    @tweet
+    respond_to do |format|
+        format.html {render :show}
+        format.json { render :show, status: :ok, location: @tweet }
```

Comment:
You need to respond using the json schema you defined in your specs otherwise your test are going to fail